themes/digital.gov/src/scss/new/_card-elsewhere.scss

```diff
@@ -1,58 +1,54 @@
 @use "uswds-core" as *;
 
-.card-elsewhere {
-  & > div {
-    @include u-display("flex");
-    @include u-flex("justify-start");
-    @include u-flex("align-start");
-  }
+.card.card-elsewhere {
```

Comment:
Added `.card` for specificity since we always want two columns (specifically mobile).